packages/@aws-cdk/aws-ecs/test/ec2/integ.event-task.lit.expected.json

```diff
@@ -1,1162 +1,1170 @@
 {
   "Resources": {
-    "Vpc8378EB38": {
-      "Type": "AWS::EC2::VPC",
-      "Properties": {
-        "CidrBlock": "10.0.0.0/16",
-        "EnableDnsHostnames": true,
-        "EnableDnsSupport": true,
-        "InstanceTenancy": "default",
-        "Tags": [
-          {
-            "Key": "Name",
-            "Value": "aws-ecs-integ-ecs/Vpc"
+      "Vpc8378EB38": {
+          "Type": "AWS::EC2::VPC",
```

Comment:
Looks like you changed the spacing to four spaces instead of two? See `a357bdef`